cmd/daemonclient/main.go

```diff
@@ -54,11 +54,11 @@ var (
 	commands = map[string]*commandOpts{
 		"wal-push": {
```

Comment:
I suggest adding the config setting for controlling this behavior. And for daemon mode, it might be the only option since it breaks backward compatibility if we change the protocol like this. I don't see the use case where we need to decide it for each WAL file anyway.

Reply:
If this setting is set via env variable, will it be merged with the rest of the config? If so, it could be a nice option